debian/st2chatops.service

```diff
@@ -14,6 +14,8 @@ TimeoutSec=60
 PrivateTmp=true
 Restart=on-failure
 RestartSec=5
+StandardOutput=syslog+console
+StandardError=syslog+console
```

Comment:
Does that fix hubot logging bug described here: https://github.com/StackStorm/st2-packages/issues/300 ?

If so, please update `st2chatops.service` for rpm as well.